### PR TITLE
Add modal with additional per task intructions

### PR DESCRIPTION
**Pull request by Era-Dorta**
This PR adds a modal at the start of the task with extra instructions that can be specified in the spec. The instructions are shown in a list.

The modal is shown at task start and also every time the "Watch tutorial video" is clicked.

### CODE REVIEW

File: covfee/client/tasks/continuous_annotation/index.tsx [hunk 1]
```diff
@@ -99,6 +99,11 @@ const ContinuousAnnotationTask: React.FC<Props> = (props) => {
       needs_upload: false,
     })
 
+  const [showInitialTaskIntructions, setshowInitialTaskIntructions] = useState(
```
Comment (Kenneth-Funes-TU-Delft) [inline covfee/client/tasks/continuous_annotation/index.tsx:102]: For consistency, please use camel case -> `setShowInitialTaskIntructions`

File: covfee/client/tasks/continuous_annotation/index.tsx [hunk 2]
```diff
@@ -645,6 +650,33 @@ const ContinuousAnnotationTask: React.FC<Props> = (props) => {
 
   return (
     <form>
+      {showInitialTaskIntructions && (
+        <Modal
+          title={"Task instructions"}
```
Comment (Kenneth-Funes-TU-Delft) [inline covfee/client/tasks/continuous_annotation/index.tsx:655]: The task already has an "instructions" panel. So would be good to use another word here: perhaps "Reminder", or "Overview" or "variant" ... not sure what is more appropriate.

File: covfee/client/tasks/continuous_annotation/spec.ts [hunk 3]
```diff
@@ -20,6 +20,7 @@ export interface ContinuousAnnotationTaskSpec extends BaseTaskSpec {
   media: MediaSpec[]
   annotations: AnnotationDataSpec[]
   prolificCompletionCode?: string
+  taskIntroInstructions?: string[]
```
Comment (Kenneth-Funes-TU-Delft) [inline covfee/client/tasks/continuous_annotation/spec.ts:23]: I did not quite understand why a list was required. Perhaps renaming to `shortInstructionsPopupBulletPoints` could help ?

**Review by Kenneth-Funes-TU-Delft — APPROVED** [review]
Looks good! I made some non-blocking recommendations.

File: covfee/client/tasks/continuous_annotation/spec.ts (hunk 3 above)
Comment (Era-Dorta) [reply covfee/client/tasks/continuous_annotation/spec.ts]: Fixed in 0772750d37dc1dcc5c873945250f37d1afe68936

File: covfee/client/tasks/continuous_annotation/index.tsx (hunk 2 above)
Comment (Era-Dorta) [reply covfee/client/tasks/continuous_annotation/index.tsx]: Fixed in 0772750d37dc1dcc5c873945250f37d1afe68936

File: covfee/client/tasks/continuous_annotation/index.tsx (hunk 1 above)
Comment (Era-Dorta) [reply covfee/client/tasks/continuous_annotation/index.tsx]: Fixed in 0772750d37dc1dcc5c873945250f37d1afe68936

**Pull request merged**